src/promptflow-tools/promptflow/tools/aoai.py

```diff
@@ -135,29 +143,16 @@ def chat(
             "frequency_penalty": float(frequency_penalty),
             "logit_bias": logit_bias,
             "user": user,
-            "headers": {"ms-azure-ai-promptflow-called-from": "aoai-tool"}
+            "extra_headers": {"ms-azure-ai-promptflow-called-from": "aoai-tool"}
```

Comment:
What is the difference between `extra_headers` and `headers`?

Reply:
How can we test to make sure the change will not break telemetry collection?

Reply:
tested.
Execute in [[Web](https://dataexplorer.azure.com/clusters/viennause2/databases/Vienna?query=H4sIAAAAAAAAA2WPP2/CMBTE934Kb4AUVwlJShgYIv5ImUChnauH/RQMiW2eX0iHfvi6Q6dOd7rhfncncoPnQ++mFh8jBg6N3bouPNVnC9MOGObQuXl2XSxevsV0RUKx/2K0GvXODGiDcfa1dY7b0TZabDZitix1sVLlWl4gR1mkpZaVeqtkmqkcqmKdr6CYxTaGO4osTaP15G6oWBI60kjiRKhMwPdIODMMPqmPdXMeL0GR8RyRjU5E7c2vHD0S/GVb6HvUh/gqER8Bqe7QcvJ/8g/+DfGS+AAAAA==)] [[Desktop](https://viennause2.kusto.windows.net/Vienna?query=H4sIAAAAAAAAA2WPP2/CMBTE934Kb4AUVwlJShgYIv5ImUChnauH/RQMiW2eX0iHfvi6Q6dOd7rhfncncoPnQ++mFh8jBg6N3bouPNVnC9MOGObQuXl2XSxevsV0RUKx/2K0GvXODGiDcfa1dY7b0TZabDZitix1sVLlWl4gR1mkpZaVeqtkmqkcqmKdr6CYxTaGO4osTaP15G6oWBI60kjiRKhMwPdIODMMPqmPdXMeL0GR8RyRjU5E7c2vHD0S/GVb6HvUh/gqER8Bqe7QcvJ/8g/+DfGS+AAAAA==&web=0)] [[cluster('viennause2.kusto.windows.net').database('Vienna')](https://dataexplorer.azure.com/clusters/viennause2/databases/Vienna)]
PromptFlowRequestsInCogsvc_RawData(ago(1h))
| where ExtendedDimension.RootRunId == '25d47c59-ba3e-405d-8c68-01c3a84937a4'
| take 100
| project-reorder PreciseTimeStamp,AOAISubscriptionId, ApiId, OperationId, CalledFrom, UserAgent, ExtendedDimension